bokeh/charts/_chartobject.py

```diff
@@ -109,6 +109,8 @@ def __init__(self, title, xlabel, ylabel, legend,
         self.__palette = palette
         self.__xgrid = xgrid
         self.__ygrid = ygrid
+        self.doc = doc
```

Comment:
No. At least for now... this is a kind of a hack to make charts share the same document/session.... I'd love to see this simplified in the next releases.